_data/simple-icons.json

```diff
@@ -5607,8 +5607,8 @@
         },
         {
             "title": "Microsoft Azure",
-            "hex": "0089D6",
-            "source": "https://developer.microsoft.com"
+            "hex": "0078D4",
+            "source": "https://portal.azure.com"
```

Comment:
While I'm reviewing the SVG - could you please update the source URL to the GitHub URL you used for this new version?